docs/Common_Files/Meadow_F7_Micro_Pinout.svg

```diff
@@ -211,7 +211,7 @@
                 <g id="D07-Label" transform="translate(35.000000, 0.000000)">
                     <rect id="Rectangle" stroke="#14607F" fill="#F8FAE4" x="0.5" y="0.5" width="49" height="20.5" rx="4"></rect>
                     <text id="Do7" font-family="MrsEavesOT-Roman, Mrs Eaves OT" font-size="22" font-weight="normal" line-spacing="20" fill="#1A80AA">
-                        <tspan x="6.5" y="16">Do7</tspan>
+                        <tspan x="6.5" y="16">DO7</tspan>
```

Comment:
Thanks for pointing this out! I believe this is a capital `O` and not a `0`.  I fixed it with a `0` here https://github.com/WildernessLabs/Documentation/pull/175